### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by eserengo**
Description
For the fourth project on your portfolio website, you will:
- Use media queries to make sure that your website displays correctly on desktop screens.
- You have time for this project scheduled for 2 days. Please ask for a code review when you are ready.

General requirements:
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used the correct ([Github flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

HTML/CSS requirements:
Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Project requirements:
IMPORTANT NOTE: For each Microverse project, you need to create a separate pull request. [Read how to work with multiple pull requests in your one repository in the most efficient way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/multiple_pull_requests.md).

You need to make sure that your desktop version looks like the one in the Figma design template that you chose. We include a screenshot of Template 1 below for reference, but you should use the template that you chose.

IMPORTANT NOTE some Figma design templates include two input fields (for first name and last name) in the desktop version and one input field (for full name) in the mobile version (look for those cases in Contact me or Get in touch sections). Please ignore this part of the design template and use only one input for the full name for all versions.

- Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider).

- In order to lay out the elements on the page you should use Grid and Flexbox.

- You don't need to implement any functionality that requires JavaScript.

- You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -22,9 +23,13 @@ img {
 
 .header {
   order: 1;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
```
Comment (Ugboaja-Uchechi) [inline style.css:20]: - [x] Good job with your design kindly make your design fills the webpage and looks as close as possible to the Figma design. Currently your desktop site isn't responsive screen size more than `768px`kindly fix this 🙏 

Your Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 2]
```diff
@@ -486,9 +498,89 @@ img {
...
+
+  .icon {
+    display: block;
+    margin: auto;
+  }
+
+  .work {
+    display: flex;
+    flex-direction: row;
+  }
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [x] Kindly make sure your projects section is centred and looks as close as possible to the Figma design 🙏 
- [x] Kindly make sure your paragraph and text doesn't look squished together please give it the appropriate padding and spacing 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 3]
```diff
@@ -486,9 +498,89 @@ img {
...
+
+  #about-myself {
+    display: flex;
+    flex-flow: row wrap;
+  }
+
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [x]  Kindly make sure your about section is centred and looks as close as possible to the Figma design 🙏
- [x] Please make sure you give the about section a `border-top-right-radius` as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 4]
```diff
@@ -486,9 +498,89 @@ img {
...
+    height: 102px;
+  }
+
+  .skill-list {
+    margin: 1em 1em 1em 4em;
+  }
+
+  .footer {
+    display: flex;
+    align-items: center;
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [x] Great job with your styling 👍 . Kindly make sure the paragraph in your contact me section has the appropriate width and alignment as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 5]
```diff
@@ -22,9 +23,13 @@ img {
...
 .header {
   order: 1;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
 }
 
-.toolbar > nav {
+.mobile-toolbar > nav {
```
Comment (Ugboaja-Uchechi) [inline style.css:23]: - [x] Kindly make sure you use the appropriate background image for the main section as seen in the figma design 🙏 
- [x] Good job 👍. Kindly make sure the content of your main section are given the appropriate margin and padding to align your content as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 6]
```diff
@@ -486,9 +498,89 @@ img {
...
+  }
+
+  #about-myself-bodytext {
+    width: 447px;
+    height: 102px;
+  }
+
+  .skill-list {
+    margin: 1em 1em 1em 4em;
+  }
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [x] Please make sure the language list are stacked side by side as seen in the Figma design 🙏 
- [x] Kindly give the `languages`, `skill`, and `framework` a border as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @eserengo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (eserengo) [reply style.css:20]: Done!

File: style.css (hunk 2 above)
Comment (eserengo) [reply style.css]: Done!

File: style.css (hunk 3 above)
Comment (eserengo) [reply style.css]: Done!

File: style.css (hunk 5 above)
Comment (eserengo) [reply style.css:23]: Done!

File: style.css (hunk 4 above)
Comment (eserengo) [reply style.css]: Done!

File: style.css (hunk 6 above)
Comment (eserengo) [reply style.css]: Done!

**Comment by eserengo** [comment]
Thank you @Ugboaja-Uchechi for your valuable feedback!!
All requested changes have been done. 😄

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @eserengo  , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Comment by eserengo** [comment]
Thank you @MaganaAsiati.
I have taken note of the optional suggestion.. absolutely right, I need more imagination on my commit messages! 😞

**Pull request merged**